src/components/DetailsOverlay/SearchRangeInput/index.tsx

```diff
@@ -0,0 +1,106 @@
+import styled from "@emotion/styled";
+import { palette } from "@leafygreen-ui/palette";
+import TextInput from "@leafygreen-ui/text-input";
+import { QueryParams } from "constants/queryParams";
+import { size } from "constants/tokens";
+import { useQueryParam } from "hooks/useQueryParam";
+import { FilterRow, StyledSubtitle } from "../styles";
+
+const { red } = palette;
+
+const SearchRangeInput: React.FC = () => {
+  const [lowerBound, setLowerQueryParam] = useQueryParam<number | undefined>(
+    QueryParams.LowerRange,
+    undefined
+  );
+  const [upperBound, setUpperBound] = useQueryParam<number | undefined>(
+    QueryParams.UpperRange,
+    undefined
+  );
+
+  const hasError =
+    lowerBound !== undefined && upperBound !== undefined
+      ? lowerBound > upperBound
+      : false;
```

Comment:
Although we now display an error this doesn't actually prevent the user from using these values. They are still applied to the search and are used. (Although i guess we check this during the search phase) what do you think?

Reply:
Great point i think you are right search and these associated search params don't really make sense as a shareable piece of data. Maybe they should live in context instead. I can address it in my upcoming search pr if you would like